src/CarbonAware.CLI/test/unitTests/Commands/EmissionsCommandTests.cs

```diff
@@ -0,0 +1,167 @@
+using CarbonAware.Aggregators.CarbonAware;
+using CarbonAware.CLI.Commands.Emissions;
+using CarbonAware.Model;
+using Microsoft.Extensions.Options;
+using Moq;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using System.Text.Json;
+
+namespace CarbonAware.CLI.UnitTests;
+
+[TestFixture]
+public class EmissionsCommandTests : TestBase
+{
+    [Test]
+    public async Task Run_CallsAggregatorAndWritesResults()
+    {
+        // Arrange
+        var emissionsCommand = new EmissionsCommand();
+        var invocationContext = SetupInvocationContext(emissionsCommand, "emissions");
+        var expectedLocationJSON = "\"Location\":\"eastus\"";
```

Comment:
Suggestion: This seems like error prune, given the fact that missing a escape char might not create the right json. Maybe it would be better to create the object, and use the json lib to serialize / deserialize.